perf/istio-install/setup_prometheus.sh

```diff
@@ -35,7 +35,7 @@ function install_prometheus() {
     kubectl wait --for=condition=Established -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
     kubectl wait --for=condition=Established -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
 
-    kubectl apply --namespace istio-prometheus -f "${PROM_OP}"
+    kubectl apply -f "${PROM_OP}"
```

Comment:
which namespace is it using? is there a namespace context set or namespace is already baked into the docs?

Reply:
I just checked the setup script, we do not have a namespace context set. This namespace should be `istio-prometheus`. However, in prom-operator.yaml file, we also have some objects with `kube-system` as namespace. That's why when we execute this command, there will be error: the namespace from the provided object "kube-system" does not match the namespace "istio-prometheus". Since Grafana is relying on  `istio-prometheus` namespace, the best solution would be using istio/installer to install prometheus operator. I will fix it later on. Currently, I will take 1.3 release performance degradation issue as the priority.